pkg/globalconfig/global_config.go

```diff
@@ -341,6 +364,19 @@ func WriteGlobalConfig(config GlobalConfig) error {
 # to ~/.ddev/bin/docker-compose.
 # Please don't use this unless directed to do so
 
+# messages:
+#   ticker_interval: 20 // Interval in hours to show ticker messages, 0 disables the ticker
+# Controls the display of the ticker messages.
+
+# remote_config:
+#   update_interval: 10 // Interval in hours to download the remote config
+#   remote:
+#     owner: ddev
+#     repo: remote-config
+#     ref: main
+#     filepath: remote-config.jsonc
+# Controls the download of the remote config. Please do not change.
```

Comment:
@mattstein can you please also have a look at these descriptions, thanks!

Reply:
Looks good to me!